doc/user/release-notes.md

```diff
@@ -20,9 +20,14 @@ For information about available versions, see our [Versions page](../versions).
   consistent with PostgreSQL. The strings `NaN`, and `[+-]Infinity` are
   accepted as input, to select the special not-a-number and infinity states
   of floating-point numbers.
+
 - Support CSV files with headers (`CREATE SOURCE...FORMAT CSV WITH HEADER`).
+
 - Support naming columns when creating CSV sources.
 
+- Require the `-w` / `--threads` command-line parameter. Consult the
```

Comment:
Given that this is a pretty major change, I'd move it to the top of the list. LGTM otherwise.